_inc/lib/core-api/wpcom-endpoints/memberships.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/**
+ * Memberships: API to communicate with "product" database.
+ *
+ * @since 7.2
```

Comment:
Since we're already past code freeze for 7.2, you'll most likely want to update references to 7.3 everywhere in that PR.

Reply:
Done!